### DISCUSSION TITLE
Program's icon

### DISCUSSION THREAD

**Discussion by SuperZombi** [lllyasviel/Fooocus#2002] [Ideas]
This program didn't have an icon, so I generated it. 🗿
<img src="https://github.com/lllyasviel/Fooocus/assets/75096786/2583847e-ac8e-4a01-aeec-6937b36c8e5a" height="240px">

**Comment by xhoxye** [comment]
This reminds me of Adobe Flash.
![QQ截图20240121122611](https://github.com/lllyasviel/Fooocus/assets/129571231/46da4c5c-1962-42dd-8c31-4f1ed4c2f9e4)

**Reply by eddyizm** [reply]
I would go back and say it reminds me of Macromedia Flash :-)
